snuba/settings/__init__.py

```diff
@@ -73,6 +73,7 @@
 
 # Clickhouse Options
 CLICKHOUSE_MAX_POOL_SIZE = 25
+CLICKHOUSE_STARTUP_CHECK_RETRIES = 60
```

Comment:
Are you planning to override this value? The default seems ok to me since 1 minute is a long time... and consumers won't make any progress if they can't connect anyway.

I wonder if we actually need to move it here and make settings more complex if there's not a use case to change it.

Reply:
prob not. I removed it